portfolio/src/main/webapp/script.js

```diff
@@ -13,16 +13,73 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * Adds a random fact to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function revealFact() {
+  // Helper function to get a random fact.
+  function getFact() {
+    const facts = [
+      'I went viral on TikTok for ski dance videos.', 'I love baking.',
+      'My favourite chocolate is Caramilk.', 'I play water polo.',
+      'I went to Stanford University on exchange.',
+      'I have worked at Deloitte.', 'I have worked at Macqaurie bank.'
+    ];
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+    // Pick a random one.
+    const fact = facts[Math.floor(Math.random() * facts.length)];
+    return fact;
+  }
 
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+  const factContainer = document.getElementById('fact-container');
+
+  // Show or hide fact.
+  if (factContainer.innerHTML == '') {
+    factContainer.innerHTML = getFact();
+  }
+
+  else {
```

Comment:
[The recommended style for Javascript braces is similar to Java](https://google.github.io/styleguide/jsguide.html#formatting-nonempty-blocks).

Reply:
removed spacing